paper.md

```diff
@@ -207,10 +207,29 @@ It can either be created by users or provided by system administrators.
 
 # Case Study {#sec:study}
 
-### **Single-Node local workflow execution**
+In this section, we present three case studies of how the Popper workflow engine allows reproducing and scaling workflows easily in different compute environments.
+<!-- briefly describe each case studies here -->
+
+### **Workflow execution on the local machine**
+
+In this section, we discuss how Popper helps researchers to write and test workflows on their local machines and iterate fast.
```

Comment:
the section doesn't discuss this, so we can either remove this sentence, or add a discussion on how researchers write and test workflows, and how they can iterate quickly